uint64 coord_idx
float64 adj_val
float64 T_traj
float64 acc_perc
bool TaskSpace
